gst/nnstreamer/tensor_decoder/tensordec.c

```diff
@@ -592,23 +576,14 @@ gst_tensordec_configure (GstTensorDec * self, const GstCaps * caps)
     return FALSE;
   }
 
-  if (self->mode == DECODE_MODE_PLUGIN) {
-    switch (self->output_type) {
-      case OUTPUT_VIDEO:
-      case OUTPUT_AUDIO:
-      case OUTPUT_TEXT:
-        break;
-      default:
-        GST_ERROR_OBJECT (self, "Unsupported type %d\n", self->output_type);
-        return FALSE;
-    }
-    self->tensor_config = config;
-    self->configured = TRUE;
-    return TRUE;
+  if (self->decoder == NULL) {
+    GST_WARNING_OBJECT (self, "Decoder plugin is not yet configured.");
```

Comment:
- Comparing to the error handling at line 214, 
1.
214 GST_ERROR_OBJECT
580 GST_WARNING_OBJECT

2. error messages are
214 Decoder plugin not yet configured 
580 Decoder plugin is not yet configured

Are there any rules or convention? If not, IMHO, it is better to handle same errors with same rules.

Reply:
change to error message
```
GST_ERROR_OBJECT (self, "Decoder plugin is not yet configured.");
```